docs/reference/ml/anomaly-detection/apis/get-bucket.asciidoc

```diff
@@ -18,12 +18,10 @@ Retrieves {anomaly-job} results for one or more buckets.
 [[ml-get-bucket-prereqs]]
 == {api-prereq-title}
 
-* If the {es} {security-features} are enabled, you must have `monitor_ml`,
-`monitor`, `manage_ml`, or `manage` cluster privileges to use this API. You also
-need `read` index privilege on the index that stores the results. The
-`machine_learning_admin` and `machine_learning_user` roles provide these
-privileges. For more information, see <<security-privileges>>,
-<<built-in-roles>>, and {ml-docs-setup-privileges}.
+Requires the following privileges:
```

Comment:
I think this intro is no longer required:
```suggestion

```

Reply:
Ah, thanks for spotting it!